LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -306,3 +306,17 @@ void Utility::SanitizeJson(_Inout_ std::wstring& str)
         i++;
     }
 }
+
+/// <summary>
+/// Comparing wstrings with ignoring the case
+/// </summary>
+/// <param name="stringA"></param>
+/// <param name="stringB"></param>
+/// <returns></returns>
+bool Utility::CompareWStrings(wstring stringA, wstring stringB)
+{
+    return stringA.size() == stringB.size()
+        && equal(stringA.cbegin(), stringA.cend(), stringB.cbegin(),

```

Comment:
nit: for better readability, suggesting that we re-format this function body, especially for the `lambda` stuff you've used:

```cpp
return stringA.size() == stringB.size() &&
    equal(
        stringA.cbegin(),
        stringA.cend(),
        stringB.cbegin(),
        [](wstring::value_type l1, wstring::value_type r1) {
                return towupper(l1) == towupper(r1);
        }
    );
```